package.json

```diff
@@ -42,7 +42,7 @@
     "eject": "react-scripts eject",
     "storybook": "start-storybook",
     "build-storybook": "build-storybook -s public",
-    "build-dictionary": "./style-dictionary/generate.sh && ./style-dictionary2/generate.sh",
+    "build-dictionary": "./style-dictionary/generate.sh && style-dictionary/generate.sh",
```

Comment:
we could remove the first command,, both do the same thing here

Reply:
basically
```
    "build-dictionary": "./style-dictionary/generate.sh",
```

Reply:
🤦 yup!